x-pack/plugins/data_enhanced/public/search/ui/search_session_indicator/search_session_indicator.tsx

```diff
@@ -119,162 +119,151 @@ const backgroundSessionIndicatorViewStateToProps: {
       iconType: 'checkInCircleFilled',
-      'aria-label': i18n.translate(
-        'xpack.data.backgroundSessionIndicator.resultsLoadedIconAriaLabel',
-        {
-          defaultMessage: 'Results loaded',
-        }
-      ),
+      'aria-label': i18n.translate('xpack.data.searchSessionIndicator.resultsLoadedIconAriaLabel', {
+        defaultMessage: 'Loaded',
+      }),
       tooltipText: i18n.translate(
-        'xpack.data.backgroundSessionIndicator.resultsLoadedIconTooltipText',
+        'xpack.data.searchSessionIndicator.resultsLoadedIconTooltipText',
         {
           defaultMessage: 'Results loaded',
         }
       ),
     },
     popover: {
-      text: i18n.translate('xpack.data.backgroundSessionIndicator.resultsLoadedText', {
+      text: i18n.translate('xpack.data.searchSessionIndicator.resultsLoadedText', {
         defaultMessage: 'Results loaded',
       }),
       primaryAction: SaveButton,
-      secondaryAction: ViewBackgroundSessionsButton,
+      secondaryAction: ViewAllSearchSessionsButton,
     },
   },
-  [SessionState.BackgroundLoading]: {
+  [SearchSessionState.BackgroundLoading]: {
     button: {
       iconType: EuiLoadingSpinner,
       'aria-label': i18n.translate(
-        'xpack.data.backgroundSessionIndicator.loadingInTheBackgroundIconAriaLabel',
+        'xpack.data.searchSessionIndicator.loadingInTheBackgroundIconAriaLabel',
         {
           defaultMessage: 'Loading results in the background',
         }
       ),
       tooltipText: i18n.translate(
-        'xpack.data.backgroundSessionIndicator.loadingInTheBackgroundIconTooltipText',
+        'xpack.data.searchSessionIndicator.loadingInTheBackgroundIconTooltipText',
         {
           defaultMessage: 'Loading results in the background',
         }
       ),
     },
     popover: {
-      text: i18n.translate('xpack.data.backgroundSessionIndicator.loadingInTheBackgroundText', {
+      text: i18n.translate('xpack.data.searchSessionIndicator.loadingInTheBackgroundText', {
         defaultMessage: 'Loading in the background',
       }),
       primaryAction: CancelButton,
-      secondaryAction: ViewBackgroundSessionsButton,
+      secondaryAction: ViewAllSearchSessionsButton,
     },
   },
-  [SessionState.BackgroundCompleted]: {
+  [SearchSessionState.BackgroundCompleted]: {
     button: {
       color: 'success',
       iconType: 'checkInCircleFilled',
       'aria-label': i18n.translate(
-        'xpack.data.backgroundSessionIndicator.resultLoadedInTheBackgroundIconAraText',
+        'xpack.data.searchSessionIndicator.resultLoadedInTheBackgroundIconAraText',
         {
           defaultMessage: 'Results loaded in the background',
         }
       ),
       tooltipText: i18n.translate(
-        'xpack.data.backgroundSessionIndicator.resultLoadedInTheBackgroundIconTooltipText',
+        'xpack.data.searchSessionIndicator.resultLoadedInTheBackgroundIconTooltipText',
         {
           defaultMessage: 'Results loaded in the background',
```

Comment:
This stays the same?

Reply:
The action is still "loaded in the background"